src/pane.coffee

```diff
@@ -468,12 +473,14 @@ class Pane extends Model
     chosen = @applicationDelegate.confirm
       message: "'#{item.getTitle?() ? uri}' has changes, do you want to save them?"
       detailedMessage: "Your changes will be lost if you close this item without saving."
-      buttons: ["Save", "Cancel", "Don't Save"]
+      buttons: if multiple then ["Save", "Cancel", "Don't Save", "No For All", "Yes For All"] else ["Save", "Cancel", "Don't Save"]
 
     switch chosen
       when 0 then @saveItem(item, -> true)
       when 1 then false
       when 2 then true
+      when 3 then return 1
```

Comment:
I don't think we should use magic numbers and mix bools and numbers. Could we use string constants, or number constants, or something more descriptive?